examples/advanced/bionemo/downstream/tap/run_sim_tap.py

```diff
@@ -22,7 +22,7 @@
 # job_name = "fedavg_tap_esm1nv"
 
 simulator = SimulatorRunner(
-    job_folder=f"jobs/{job_name}", workspace=f"/tmp/nvflare/results/{job_name}", n_clients=n_clients, threads=n_clients
+    job_folder=f"jobs/{job_name}", workspace=f"/tmp/nvflare/results/{job_name}", n_clients=n_clients, threads=1
```

Comment:
why change to set "threads=1", is it because not enough resources?

Reply:
I set this to the minimum in case people do not have resources, for demo purposes people might test these on their laptops. I made a note in the notebooks about this :)

Reply:
thanks for the prompt reply, resolved.

Reply:
@YuanTingHsieh it would be great if you could test the notebooks as well, please let me know if you run into issues :)